docs/standard/serialization/system-text-json-migrate-from-newtonsoft-how-to.md

```diff
@@ -14,19 +14,69 @@ helpviewer_keywords:
+| Allow trailing commas                                 | ![check mark icon](../../../images/core/check-mark.png) [AllowTrailingCommas global setting](#trailing-commas) |
+| Custom converter registration                         | ![check mark icon](../../../images/core/check-mark.png) [Order of precedence differs](#converter-registration-precedence) |
+| Minimal character escaping                            | ![check mark icon](../../../images/core/check-mark.png) [Strict character escaping, configurable](#character-escaping) |
+| No maximum depth by default                           | ![check mark icon](../../../images/core/check-mark.png) [Default maximum depth 64, configurable](#maximum-depth) |
+| `NullValueHandling.Ignore` global setting             | ![check mark icon](../../../images/core/check-mark.png) [IgnoreNullValues global option](#omit-null-value-properties) |
+| `JToken` (like `JObject`, `JArray`)                   | ![check mark icon](../../../images/core/check-mark.png) [JsonDocument and JsonElement](#jsondocument-and-jsonelement-compared-to-jtoken-like-jobject-jarray) |
+| `JsonTextReader`                                      | ![check mark icon](../../../images/core/check-mark.png) [Utf8JsonReader](#utf8jsonreader-compared-to-jsontextreader) |
+| `JsonTextWriter`                                      | ![check mark icon](../../../images/core/check-mark.png) [Utf8JsonWriter](#utf8jsonwriter-compared-to-jsontextwriter) |
+| Support for a broad range of types                    | ![warning icon](../../../images/core/yellow-warning.png) [Some types require custom converters](#types-without-built-in-support) |
+| Deserialize inferred type to `object` properties      | ![warning icon](../../../images/core/yellow-warning.png) [Not supported, workaround, sample](#deserialization-of-object-properties) |
+| `DateTimeZoneHandling`, `DateFormatString` settings   | ![warning icon](../../../images/core/yellow-warning.png) [Not supported, workaround, sample](#specify-date-format) |
+| Deserialize strings as numbers                        | ![warning icon](../../../images/core/yellow-warning.png) [Not supported, workaround, sample](#quoted-numbers) |
+| Deserialize `Dictionary` with non-string key          | ![warning icon](../../../images/core/yellow-warning.png) [Not supported, workaround, sample](#dictionary-with-non-string-key) |
+| Polymorphic serialization                             | ![warning icon](../../../images/core/yellow-warning.png) [Not supported, workaround, sample](#polymorphic-serialization) |
+| Polymorphic deserialization                           | ![warning icon](../../../images/core/yellow-warning.png) [Not supported, workaround, sample](#polymorphic-deserialization) |
+| `Required` setting on [JsonProperty] attribute        | ![warning icon](../../../images/core/yellow-warning.png) [Not supported, workaround, sample](#required-properties) |
```

Comment:
Consistency with others:
```suggestion
| `Required` setting on `[JsonProperty]` attribute        | ![warning icon](../../../images/core/yellow-warning.png) [Not supported, workaround, sample](#required-properties) |
```

Also, consider adding a sentence referencing the attribute there - In `Newtonsoft.Json` you would specifiy that a property is required by setting `Required` on the the `[JsonProperty]` attribute.